CHANGELOG.md

```diff
@@ -76,6 +76,10 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - `exonum_derive` crate has been added with custom derives for `ProtobufConvert`
   and `TransactionSet`. (#1055)
 
+- `TransactionResult` is now serialized using protobuf. Empty description
```

Comment:
I'd like to give bigger accent on the fact that we don't allow empty descriptions now than the fact that it uses protobuf now.

Reply:
I think its fine, its on the same line.